egs2/chime7_task1/asr1/run.sh

```diff
@@ -142,34 +141,32 @@ if [ ${stage} -le 2 ] && [ ${stop_stage} -ge 2 ]; then
     local/run_gss.sh --manifests-dir $manifests_root --dset-name $dset_name \
           --dset-part $dset_part \
           --exp-dir $gss_dump_root \
-          --cmd $cmd_gss \
+          --cmd "$cmd_gss" \
           --nj $ngpu \
           --max-segment-length $max_segment_length \
           --max-batch-duration $gss_max_batch_dur \
           --channels $channels \
-          --use-selection $use_selection
+          --use-selection $use_selection \
+          --top-k $top_k
     log "Guided Source Separation processing for ${dset_name}/${dset_part} was successful !"
   done
 fi
 
 
 if [ ${stage} -le 3 ] && [ ${stop_stage} -ge 3 ]; then
-
-  asr_train_set=kaldi/train_all_mdm_ihm_rvb_gss
-  asr_cv_set=kaldi/chime6/dev/gss # use chime only for validation
   # Decoding on dev set because test is blind for now
-  # NOTE that ESPNet will not make copies of the original Kaldi manifests
-  # e.g. for training and cv, so if you set $train_max_segment_length these
-  # will be discarded also from the test set (if the test set is the same as evaluation)
-  # you need to make a copy !
-  ./utils/copy_data_dir.sh data/kaldi/chime6/dev/gss data/kaldi/chime6/dev/gss_inf
-  asr_tt_set="kaldi/chime6/dev/gss_inf kaldi/dipco/dev/gss/ kaldi/mixer6/dev/gss/"
 
   pretrained_affix=
   if [ -n "$use_pretrained" ]; then
-    asr_train_set=kaldi/dipco/dev/gss # dummy one, it is not used
+    asr_train_set=""#kaldi/dev_ihm_all # dummy one, it is not used
```

Comment:
```suggestion
    asr_train_set="" #kaldi/dev_ihm_all # dummy one, it is not used
```

asr_train_set will be "#kaldi/dev_ihm_all" thought it's no problem

Reply:
Thanks, yeah now it is ignored but definitely a space was supposed to be there